keps/sig-storage/2644-honor-pv-reclaim-policy/README.md

```diff
@@ -197,8 +208,22 @@ Once the volume is deleted from the backend, the finalizer can be removed. This
 
 Note: This feature should work with CSI Migration disabled or enabled.
 
+#### Statically Provisioned volumes
+
+The finalizer `external-provisioner.volume.kubernetes.io/finalizer` is added only to statically provisioned volumes that are associated with a PVC.
```

Comment:
Will PV controller add the finalizer? Why is finalizer `kubernetes.io/pv-controller` not used here?

IMO, the finalizers should be the same as with dynamically provisioned volumes. I would not event mention it as a separate chapter, I would mention that statically provisioned volumes will behave the same as dynamically provisioned ones, except for "PV is not (yet?) associated with a PVC".

Reply:
Since it is a CSI driver volume `external-provisioner.volume.kubernetes.io/finalizer` is added, this is added by the external-provisioner. `kubernetes.io/pv-controller` is added to in-tree volumes.

Reply:
I've addressed the comment, removed the statically and dynamically provisioned volumes section, and merged them into one as you specified.